Template/Template.csproj

```diff
@@ -32,16 +32,10 @@
         <PackageReference Include="UnityEngine.Modules" Version="2022.3.9" IncludeAssets="compile"/>
     </ItemGroup>
 
-    <ItemGroup>
-        <!-- https://github.com/EvaisaDev/UnityNetcodePatcher -->
-        <!-- Allows to patch elements like networked behaviours, RPCs, etc. -->
-        <!-- Also, without that, the CI/CD will fail because the game's assemblies are not findable -->
-        <PackageReference Include="Evaisa.NetcodePatcher.MSBuild" Version="3.*" PrivateAssets="all"/>
-        <NetcodePatch Include="$(TargetPath)"/>
-    </ItemGroup>
-
     <!-- https://github.com/EvaisaDev/UnityNetcodePatcher#usage-as-a-post-build-event -->
     <!-- Syntax to use the tool installed globally -->
+    <!-- Allows to patch elements like networked behaviours, RPCs, etc. -->
+    <!-- Also, without that, the CI/CD will fail because the game's assemblies are not findable -->
```

Comment:
This comment is misleading - it is actually [this line](https://github.com/Distractic/LethalCompanyTemplate/blob/cd2660489ffa13b844d70df9c0b203114b3f2eef/Common.props#L58) that is responsible for referencing the game's assemblies in CI environments

Reply:
Ah true, I mixed it up a bit